docker/Dockerfile-php72

```diff
@@ -0,0 +1,10 @@
+FROM php:7.2-cli
+
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+
+RUN chmod +x /usr/local/bin/install-php-extensions && \
+    install-php-extensions pcntl redis mongodb-1.4.4 && \
```

Comment:
This one requires a newer mongodb extension. 1.1.9 doesn't work with 7.2 and 1.4.4 is the last version that works with MongoDB 2.6